CONTRIBUTING.adoc

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright (c) 2018 Contributors to the Eclipse Foundation
```

Comment:
very minor, but probably should be 2019?

Reply:
Changed to 2019